plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabProjectAccessTokenAction.ts

```diff
@@ -14,11 +14,14 @@
  * limitations under the License.
  */
 
-import { createTemplateAction } from '@backstage/plugin-scaffolder-node';
+import { InputError } from '@backstage/errors';
 import { ScmIntegrationRegistry } from '@backstage/integration';
-import commonGitlabConfig from '../commonGitlabConfig';
-import { getToken } from '../util';
+import { createTemplateAction } from '@backstage/plugin-scaffolder-node';
+import { AccessTokenScopes } from '@gitbeaker/core';
```

Comment:
This doesn't look like it comes from `@gitbeaker/core`? It might need bumping to a later version in the package.json?

Reply:
I used version 39.25.0 from @gitbeaker/core. However, there seems to have been a breaking change in the types between versions 35 and 39, so that there are problems with the actions gitlabMergeRequest.ts, gitlabRepoPush.ts and helpers.ts even after the update to the version.

> yarn build:api-reports

TIP: You can generate api-reports for select packages by passing package paths:

       yarn build:api-reports packages/config packages/core-plugin-api plugins/*

# Compiling TypeScript
plugins/scaffolder-backend-module-gitlab/src/actions/gitlabMergeRequest.ts:22:10 - error TS2305: Module '"@gitbeaker/core"' has no exported member 'Types'.

22 import { Types } from '@gitbeaker/core';
            ~~~~~

plugins/scaffolder-backend-module-gitlab/src/actions/gitlabRepoPush.ts:22:10 - error TS2305: Module '"@gitbeaker/core"' has no exported member 'Types'.

22 import { Types } from '@gitbeaker/core';
            ~~~~~

plugins/scaffolder-backend-module-gitlab/src/actions/helpers.ts:20:10 - error TS2305: Module '"@gitbeaker/core"' has no exported member 'Resources'.

20 import { Resources } from '@gitbeaker/core';
            ~~~~~~~~~


Found 3 errors in 3 files.

Errors  Files
     1  plugins/scaffolder-backend-module-gitlab/src/actions/gitlabMergeRequest.ts:22
     1  plugins/scaffolder-backend-module-gitlab/src/actions/gitlabRepoPush.ts:22
     1  plugins/scaffolder-backend-module-gitlab/src/actions/helpers.ts:20

Reply:
I think you'll need to check the breaking changes and see if you can update those other files with the latest imports.